src/common/components/repository-input/index.js

```diff
@@ -48,6 +50,24 @@ export class RepositoryInput extends Component {
     const isTouched = input.inputValue.length > 2;
     const isValid = !!input.repository && !input.error;
 
+    let submitButton;
+    if (conf.get('LP_API_USERNAME')) {
+      // Main path, for use in production.
+      submitButton = (
+        <Button type='submit' disabled={!isValid || input.isFetching}>
+          { input.isFetching ? 'Creating...' : 'Create' }
+        </Button>
+      );
+    } else {
+      // If the Launchpad API isn't configured, fall back to just verifying
+      // the repository.  This is handy in development.
+      submitButton = (
+        <Button type='submit' disabled={!isValid || input.isFetching}>
+          { input.isFetching ? 'Verifying...' : 'Verify' }
+        </Button>
+      );
+    }
```

Comment:
Yes, it would be good to separate those, maybe even into smaller components when it makes sense, but at this point we kind of discover how flow of the UI should look like, so let's keep it all here as simple as possible and we will refactor once we have everything in place.